wpeverest/user-registration
# Enhance - Introduce tooltips in Form Builder

### All Submissions:






### Changes proposed in this Pull Request:
This PR introduces tooltips in the form builder.

### How to test the changes in this Pull Request:

1. Switch to this branch. Also checkout to the branch `enhance/tooltip-works` for the following addons:
- User Registration Conditional Logic
- User Registration Field Visibility
- User Registration File Upload
- User Registration Advanced Fields
- User Registration Mailchimp
- User Registration Multi-Part
- User Registration Payments
- User Registration PDF Form Submission
- User Registration Stripe
- User Registration WooCommerce
- User Registration Learndash
2. Go to form builder
3. See if all tooltips are behaving properly in `Field Settings` and `Form Settings`

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Introduce tooltips in Form Builder
